### PR TITLE
Move go application in submodule

### PR DESCRIPTION
**Pull request by Jasper-Ben**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "go-k8s-userland"]
+	path = go-k8s-userland
+	url = https://github.com/ovgu-cs-workshops/go-k8s-userland.git
```
Comment (martin31821) [inline .gitmodules]: recursive submodule please.

**Review by martin31821 — CHANGES_REQUESTED** [review]
Please remove old go code from here.

File: .gitmodules (hunk 1 above)
Comment (Jasper-Ben) [reply .gitmodules]: Done.

**Comment by Jasper-Ben** [comment]
> Please remove old go code from here.

Done.

**Review by martin31821 — APPROVED** [review]

**Pull request merged**